web/components/templates/requestsV2/builder/requestBuilder.tsx

```diff
@@ -194,9 +194,17 @@ const isAssistantRequest = (request: HeliconeRequest) => {
 
 const getNormalizedRequest = (request: HeliconeRequest): NormalizedRequest => {
   try {
-    return getRequestBuilder(request).build();
+    const normalizedRequest = getRequestBuilder(request).build();
```

Comment:
This is not solving the issue. If the model in the images correctly shows in the request drawer, it should also show correctly in the request page. There is another issue here.